test/performance/benchmarks/scale-from-zero/main.go

```diff
@@ -360,5 +360,12 @@ func checkSLA(results *vegeta.Metrics, p95min time.Duration, p95max time.Duratio
 		return fmt.Errorf("SLA 2 failed. Max latency is higher than %dms: %s", latencyMax, results.Latencies.Max)
 	}
 
+	// SLA 3: making sure the defined vegeta rates/total requests are met
+	if results.Requests == uint64(parallel) {
```

Comment:
Here it seems that:

```parallelCount = flag.Int("parallel", 0, "The count of ksvcs we want to run scale-from-zero in parallel")``` should be equal to `results.Requests`?
Are we sending just one request from scaling from zero?
If so could we add a better var name there or a more detailed comment?